MAINTAINERS.md

```diff
@@ -0,0 +1,2 @@
+Stefan Berger <stefanb@linux.ibm.com>
```

Comment:
I suggest using a similar format to the project maintainers file and maybe adding something like this to the top
```
# imgcrypt maintainers
#
# As a containerd sub-project, containerd maintainers are also included from https://github.com/containerd/project/blob/master/MAINTAINERS.
# See https://github.com/containerd/project/blob/master/GOVERNANCE.md for description of maintainer role
#
# MAINTAINERS
# GitHub ID, Name, Email address
```

Reply:
done